src/message/messageActionSheet.js

```diff
@@ -230,9 +247,10 @@ export const showActionSheet = (
       ...params,
     });
   };
+  const decorateTitle = title => title.replace('%user%', params.message.sender_full_name);
```

Comment:
This would be handled better where the title is assigned, for the sake of clarity.

Reply:
I agree with you here. The most elegant solution here is to use LocalizableString. The problem is that this must be a string and `_` does not replace variables either.